chia/cmds/coin_funcs.py

```diff
@@ -88,7 +89,7 @@ def print_coins(
                 break
             coin, conf_height = coins[i + j]
             address = encode_puzzle_hash(coin.puzzle_hash, addr_prefix)
-            amount_str = print_balance(coin.amount, mojo_per_unit, addr_prefix, decimal_only=True)
+            amount_str = print_balance(coin.amount, mojo_per_unit, "", decimal_only=True)
```

Comment:
right. the address prefix doesn't tell us what kind of asset this is. We *could* have a lookup table to recognize known assets though. But it makes sense to just not say anything for now